### PR TITLE
Monorepo refactor

### PR DESCRIPTION
**Pull request by dadiorchen**
* A problem: I removed husky because I don't know how to set it up for monorepo.
* I tested the web version, and the web mode in Expo, both work. (I can not run simulator)

### CODE REVIEW

File: apps/web/pages/home.tsx [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+import ArrowIcon from '@mui/icons-material/ArrowForward';
+import HomeIcon from '../images/home.svg';
+import V3Icon from '../images/v3.svg';
+import V4Icon from '../images/v4.svg';
+import OfferIcon from '../images/offer.svg';
+import { useRouter } from 'next/router';
+import { useRecoilState } from 'recoil';
+import currentUser from '../states/currentUser';
+import { useEffect, useState } from 'react';
+import * as accounts from 'demo-core/models/api/accounts';
```
Comment (dadiorchen) [inline apps/web/pages/home.tsx:32]: Use the model

File: package.json [hunk 2]
```diff
@@ -1,46 +1,12 @@
...
-    "typescript": "^4.9.4"
-  },
-  "private": true
+  "main": "index.js",
+  "repository": "git@github.com:Greenstand/treetracker-web-map-core-demo.git",
+  "author": "dadiorchen <antlet2004@gmail.com>",
+  "license": "MIT",
+  "workspaces": {
+    "packages": ["apps/*", "packages/*"]
+  }
```
Comment (dadiorchen) [inline package.json:11]: Define monorepo by yarn workspace

**Comment by dagmawig** [comment]
can we go over this change when you have time? I got some questions.

**Comment by dadiorchen** [comment]
@dagmawig you can book my time or do it on the weekly meeting. 
What's your question?

**Comment by dagmawig** [comment]
> 

@pierrelstan  explained what you are trying to accomplish with the monorepo. It makes sense now.

**Review by dagmawig — APPROVED** [review]

**Comment by dadiorchen** [comment]
well done!

**Pull request merged**